packages/lesswrong/components/search/TagsSearchHit.tsx

```diff
@@ -1,19 +1,29 @@
 import React from 'react';
-import { registerComponent } from '../../lib/vulcan-lib';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
 import { tagGetUrl } from '../../lib/collections/tags/helpers';
 import { Link } from '../../lib/reactRouterWrapper';
 import { Snippet } from 'react-instantsearch-dom';
 import type { Hit } from 'react-instantsearch-core';
+import LocalOfferOutlinedIcon from '@material-ui/icons/LocalOfferOutlined';
 
 const styles = (theme: ThemeType): JssStyles => ({
   root: {
-    marginLeft: theme.spacing.unit,
-    marginTop: 6,
-    marginBottom: theme.spacing.unit/2
+    padding: 10,
+    paddingTop: 8,
+    paddingBottom: 8,
+    display: 'flex',
+    alignItems: 'center',
+    borderTop: "solid 1px rgba(0,0,0,.1)"
```

Comment:
If there are no "user" search results, then this `border-top` will show up at the top of the dropdown:
<img width="300" alt="Screen Shot 2022-05-17 at 5 27 08 PM" src="https://user-images.githubusercontent.com/9057804/168914052-5c0e4f83-1d32-432d-aa7d-b8e86fca24e1.png">

Perhaps using `border-bottom` would work better, especially if you add the "see all results" link under these results.

Reply:
That ends up causing the same problem – the issue is that I'm handling the UsersSearchHit padding different from everything else, because user search hits are comparatively teeny, and giving them an extra 20px of padding adds up to a lot of wasted space.

So instead, the overall list of users gets a small bit of padding, but then that ends up looking awkward when it's empty. I'll see if I can figure out a solution.

Reply:
Hmm I see what you mean. I think this is fine for now, but maybe in the future it would be worth refactoring a bit to use [StateResults](https://www.algolia.com/doc/api-reference/widgets/state-results/react/). Then we could hide that extra border when there are no user results.

Reply:
Nod. I thought of some other solutions overnight but think I'd prefer to leave them for a future PR.